src/containers/NdlaFilm/components/NdlaFilmArticle.tsx

```diff
@@ -53,43 +61,58 @@ const NdlaFilmArticle = ({ fieldName, onUpdateArticle }: Props) => {
     initSelectedArticle();
   }, [field.value]);
 
-  const onSearch = useCallback((query: string, page?: number) => {
-    return searchArticles({ articleTypes: ["frontpage-article"], page, query });
-  }, []);
-
   return (
-    <>
-      <FieldHeader title={t("ndlaFilm.editor.moreInfoTitle")} subTitle={t("ndlaFilm.editor.moreInfoSubTitle")} />
+    <FieldRoot>
       {selectedArticle && (
-        <ArticleElement>
-          <Link to={toEditFrontPageArticle(selectedArticle.id, selectedArticle.title.language)}>
-            {selectedArticle.title.title}
-          </Link>
-          <IconButtonV2
-            aria-label={t("ndlaFilm.editor.removeArticleFromMoreInformation")}
-            variant="ghost"
-            title={t("ndlaFilm.editor.removeArticleFromMoreInformation")}
-            colorTheme="danger"
-            data-testid="elementListItemDeleteButton"
-            onClick={() => onUpdateArticle(field, form, null)}
-          >
-            <DeleteForever />
-          </IconButtonV2>
-        </ArticleElement>
+        <ListItemRoot variant="subtle">
```

Comment:
Her kunne du brukt `ListResource` istedet! trenger muligens bare å tilgjengeliggjøre list item root props, for å styre contexten